packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -42,6 +42,7 @@
           <va-icon
             :color="toggleIconColor"
```

Comment:
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/35e48b8f-ba1d-4eff-9c8f-c53430d74f74)
change default text to "(outline)"